packages/cli-tool/test/commands/generate/index.test.ts

```diff
@@ -1,17 +1,111 @@
+import * as fs from 'fs';
+
 import { expect, test } from '@oclif/test';
+import Inquirer from 'inquirer';
+
+import { bootstrapTestData } from '../../add-ons/ui-framework/bootstrap';
+import { tailwindCssTestData } from '../../add-ons/ui-framework/tailwind-css';
+import { gitHubTestData, gitLabTestData } from '../../add-ons/version-control';
+import { TestScenario } from '../../helpers/test-scenario';
+
+const templateRepoPath = 'file:./packages/cra-template';
+const projectName = 'test-app';
+const testFolderPath = '../../';
+
+const projectPath = `${testFolderPath}${projectName}`;
+
+const testScenarios: TestScenario[] = [
+  {
+    options: {
+      versionControl: 'github',
+      uiFramework: 'bootstrap',
```

Comment:
Not sure that we can split this by testing one option per test case and run them parallelly to save time
My idea is to have separate test folders for each option and adjust the test script command to accept a part to the test folder then on Github workflow we can use metrics that specify each test folder 💭 
@malparty @tyrro If you agree maybe we can have a separate task to work on this. 🙌🏼

Reply:
@carryall the idea is very good indeed! I did not think of async runnings! 💡 
💯 to create a new task for a later implementation. It will be easier to review and discuss on the changes! 🙌

Reply:
Add the issue [here](https://github.com/nimblehq/react-templates/issues/116)